CDP4SiteDirectory/ViewModels/Dialogs/PersonDialogViewModel.cs

```diff
@@ -92,6 +92,19 @@ public PersonDialogViewModel(Person person, ThingTransaction transaction, ISessi
                 this.RaisePropertyChanged("Password");
                 this.RaisePropertyChanged("PasswordConfirmation");
             });
+
+            this.WhenAnyValue(vm => vm.SelectedRole).Subscribe(x =>
+            {
+                this.RaisePropertyChanged(nameof(this.IsSelectedRoleDeprecated));
```

Comment:
I understand people here type property name as string, but I prefer it strongly typed to avoid typos.